cli/src/commands/squash.rs

```diff
@@ -49,7 +49,10 @@ use crate::description_util::join_message_paragraphs;
 use crate::description_util::try_combine_messages;
 use crate::ui::Ui;
 
-/// Move changes from a revision into another revision
+/// Move changes from a revision into its parent
```

Comment:
This change is specific to the `-r` option, while not mentioning the other options. It was better before.

Reply:
Moving into its parent is the default, not specific to `-r`, unless I'm missing something?

```
$ jj new
Working copy  (@) now at: nqqwywvr 0133ff96 (empty) (no description set)
Parent commit (@-)      : oopqsnxt bad794b5 (empty) (no description set)
<make change>
$ jj diff --stat
cli/src/commands/squash.rs | 1 +
1 file changed, 1 insertion(+), 0 deletions(-)
$ jj squash
Working copy  (@) now at: tqsnolsy bb609055 (empty) (no description set)
Parent commit (@-)      : oopqsnxt c424f618 (no description set)
```